Makefile

```diff
@@ -197,6 +197,6 @@ install-geth:
 	./ops/scripts/geth-version-checker.sh && \
 	 	(echo "Geth versions match, not installing geth..."; true) || \
  		(echo "Versions do not match, installing geth!"; \
- 			go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
+ 			go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat versions.json | jq -r '.geth'); \
  			echo "Installed geth!"; true)
```

Comment:
a neckbeard would say this is a useless cat because you can just do `jq -r .geth < versions.json`